### DISCUSSION TITLE
How to downgrade to previous minor version

### DISCUSSION THREAD

**Discussion by lucpotage** [nuxt/nuxt#18523] [Questions] [answered]
Hi!

I tried to return to a previous version of Nuxt but `npm run dev` still runs Nuxt 3.1.0. Even after cleaning my node_modules directory and removing package-lock.json, no success.

What am I missing?

**Comment by TheAlexLichter** [comment]
Mind sharing your project?
Did you pin Nuxt to `"nuxt": "3.0.0"` (or whatever version you want)?

**Comment by lucpotage** [comment, marked as answer]
Mmm, that was stupid. It was because of the global nuxi package installed.
When running `npm run dev`, the global script took over the local one.

Thanks for you help.
